src/main/resources/parameters/scale_factors.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0"?>
+<scale_factors>
+
+    <scale_factor name="0.003">
+        <property>
+            <name>generator.numPersons</name>
+            <value>50</value>
+        </property>
+        <property>
+            <name>generator.startYear</name>
+            <value>2010</value>
+        </property>
+        <property>
+            <name>generator.numYears</name>
+            <value>3</value>
```

Comment:
Should the value of `generator.startYear` and `generator.numYears` be same with parameter.ini?

Reply:
Set the startYear to 2020. But I leave the numYears unmodified which I think you are free to update the value depending on the data in the future.